tools/udisksctl.c

```diff
@@ -2187,12 +2187,15 @@ handle_command_power_off (gint        *argc,
           for (l = objects; l != NULL; l = l->next)
             {
               object = UDISKS_OBJECT (l->data);
-              ata = udisks_object_peek_drive_ata (object);
-              if (ata != NULL)
+              drv = udisks_object_get_drive (object);
```

Comment:
Please use `udisks_object_get_drive()` - the difference is about returned object memory ownership.

Reply:
Sorry, you mean "use `udisks_object_peek_drive()`", right?

Reply:
Ooops, yes, of course.